### ISSUE TITLE
A more real night

### ISSUE THREAD

**Issue by Pau1am** [ShaneBeeStudios/SurvivalPlus#3]
**Is your feature request related to a problem? Please describe:**
I think the vanilla version of the night is too simple. In my opinion, the player's sight will be hindered to a certain extent at night. They need to hold the light source to see more clearly at night

**Describe the solution you'd like:**
When night falls, players will be affected by a greater degree of blindness buff. When the player is holding a torch and other light sources, the blind buff will only have a small impact. In addition, if you want, you can also add some new items, such as miner hats, flashlights and other items so that players can solve the light source problem without holding items, or even completely not affected by blindness buff.

**Describe alternatives you've considered:**
not yet

**Additional context:**
Nothing

**Comment by ShaneBeee** [comment]
This is an interesting idea, and could potentially be fun to work with.

**Comment by Pau1am** [comment]
Yes, I look forward to seeing this mechanism added in this plugin in the future!
